setup.py

```diff
@@ -58,18 +58,19 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
```

Comment:
@Kriechi probably should drop 3.4 as well, also can you add to the changelog?

Reply:
3.3 was already dropped in one of the last releases - but for some reason was still mentioned here. This PR does not change the supported Python versions - it only makes it explicit in setup.py.

Yes, dropping py3.4 would have been my next PR 😄